CMakeLists.txt

```diff
@@ -493,6 +493,7 @@ target_compile_definitions(_C PRIVATE CUTLASS_ENABLE_DIRECT_CUDA_DRIVER_CALL=1)
 set(VLLM_MOE_EXT_SRC
   "csrc/moe/torch_bindings.cpp"
   "csrc/moe/moe_align_sum_kernels.cu"
+  "csrc/moe/moe_wna16.cu"
```

Comment:
We might want to put this source in a `if(VLLM_GPU_LANG STREQUAL "CUDA")` section - I'm not sure if this might accidentally get built on AMD GPUs for instance

Reply:
updated.